demo/app.py

```diff
@@ -67,15 +67,15 @@ def gen_button_clicked():
 
     col1, col2 = st.columns(2)
 
-    raw_text = DATA_LOADERS[extension](uploaded_file)
+    original_text = DATA_LOADERS[extension](uploaded_file)
```

Comment:
any reason for these changes?

Reply:
`raw_text` was shadowing from [this](https://github.com/mozilla-ai/document-to-podcast/pull/72/files/14d6aa0b488b263745592add3513d0bf23ba7e8c#diff-9cf97de43f27d5f48ea9dd7c5051bd018c246bcd5d35ec74811bbb7fff76246cL106) function